examples/nextjs/pages/_document.js

```diff
@@ -67,12 +67,7 @@ MyDocument.getInitialProps = async ctx => {
   return {
     ...initialProps,
     // Styles fragment is rendered after the app and page rendering finish.
-    styles: (
-      <React.Fragment>
-        {initialProps.styles}
-        {sheets.getStyleElement()}
-      </React.Fragment>
-    ),
+    styles: [...(initialProps.styles || []), sheets.getStyleElement()],
```

Comment:
The logic seems to still work in v9. It seems that the styled-components example on the Next.js side is still using the former approach. What do you think of sticking to it?

Reply:
tbh, I'd just like to try and make
```js
styles: [
  ...initialProps.styles, 
  sheets.getStyleElement()
],
```
work. But that'll involve some changes in `next.js`.